apps/web/src/app/[locale]/challenge/_components/comments/delete/index.tsx

```diff
@@ -28,12 +28,19 @@ export function CommentDeleteDialog({
 
   async function handleDeleteComment() {
```

Comment:
can we ensure this delete button also appears for admins and mods. its currently only for author i believe

Reply:
yes i was about to ask that